test/kernel_acl.js

```diff
@@ -375,13 +375,23 @@ contract('Kernel ACL', accounts => {
                     })
                 })
 
-                it('fails petrifying existing permission by no manager', async () => {
+                it('fails burning existing permission by no manager', async () => {
                     // create permission
                     await acl.createPermission(granted, kernelAddr, MOCK_ROLE, granted, { from: permissionsRoot })
 
                     return assertRevert(async () => {
-                        // try to lock it
-                        await acl.lockPermission(kernelAddr, MOCK_ROLE, { from: noPermissions })
+                        // try to burn it
+                        await acl.burnPermissionManager(kernelAddr, MOCK_ROLE, { from: noPermissions })
+                    })
+                })
+
+                it('fails trying to create a burned permission which already has a manager', async () => {
+                    // create permission
+                    await acl.createPermission(granted, kernelAddr, MOCK_ROLE, granted, { from: permissionsRoot })
+
+                    return assertRevert(async () => {
+                        // try to create it burnt
+                        await acl.createBurnedPermission(kernelAddr, MOCK_ROLE, { from: permissionsRoot })
```

Comment:
I'd add another test either here, or separately, where we then revoke the only grantee of the permission and test that `acl.createBurnedPermission()` still fails.